src/app/reducers/vm/redux/vm-creation.effects.ts

```diff
@@ -622,7 +622,8 @@ export class VirtualMachineCreationEffects {
       params.hypervisor = 'KVM';
     }
 
-    if (securityGroups && securityGroups.length && securityGroups[0].id) {
+    const isSecurityGroupSelected = securityGroups && securityGroups.length && securityGroups[0].id;
+    if (isSecurityGroupSelected && state.zone.securitygroupsenabled) {
```

Comment:
is it still possible to select security group event if the zone is not `securitygroupsenabled` ?

Reply:
No

```  
public showSecurityGroups(): boolean {
    return (
      this.vmCreationState.zone &&
      this.vmCreationState.zone.securitygroupsenabled &&
      this.isSecurityGroupEnabled
    );
  }
```